deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -3042,13 +3042,30 @@ spec:
                 - containerPort: 8081
                 resources: {}
                 securityContext:
-                  privileged: true
+                  privileged: false
+                  runAsNonRoot: true
+                volumeMounts:
+                - mountPath: /etc/ceph/ceph.conf
+                  name: ceph-config
+                  subPath: ceph.conf
+                - mountPath: /etc/ceph/keyring
+                  name: ceph-config
+                  subPath: keyring
               serviceAccountName: ocs-metrics-exporter
               tolerations:
               - effect: NoSchedule
                 key: node.ocs.openshift.io/storage
                 operator: Equal
                 value: "true"
+              volumes:
+              - configMap:
+                  items:
+                  - key: ceph.conf
+                    path: ceph.conf
+                  - key: keyring
+                    path: keyring
+                  name: ocs-metrics-exporter-ceph-conf
```

Comment:
no need to mount each keys, just mount the whole configmap like this https://github.com/ceph/ceph-csi/commit/cc6c51395ebb41550a535caf952abc656caf44f6?